kolibri/content/management/commands/generate_schema.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+import pickle
+
+from django.apps import apps
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from kolibri.content.utils.sqlalchemybridge import SCHEMA_PATH_TEMPLATE, get_default_db_string
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import sessionmaker
+
+DATA_PATH_TEMPLATE = os.path.join(os.path.dirname(__file__), '../fixtures/{name}_content_data.json')
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('version', type=str)
+
+    def handle(self, *args, **options):
```

Comment:
I'm not sure what the standard is for management commands, but can you add a description of how to use this management command, and the motivations for it?

Reply:
Can do!